packages/edit-navigation/src/store/selectors.js

```diff
@@ -81,3 +81,14 @@ export const getMenuItemForClientId = createRegistrySelector(
 		return select( coreStore ).getMenuItem( mapping[ clientId ] );
 	}
 );
+
+/**
+ * Returns true if the inserter is opened.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {boolean} Whether the inserter is opened.
+ */
+export function isInserterOpened( state = false ) {
+	return !! state.blockInserterPanel;
```

Comment:
Should this be conditional?
```suggestion
	return !! state?.blockInserterPanel;
```

Reply:
The reducer defines a default state value of `false` so I think we're ok here but there's no harm so I've updated.